applications/dashboard/models/class.usermodel.php

```diff
@@ -941,7 +941,6 @@ public function joinUsers(&$Data, $Columns, $Options = array()) {
 
         // Join the user data using prefixes (ex: 'Name' for 'InsertUserID' becomes 'InsertName')
         $Join = val('Join', $Options, array('Name', 'Email', 'Photo'));
-        $UserPhotoDefaultUrl = function_exists('UserPhotoDefaultUrl');
```

Comment:
This should be resolved with this PR: https://github.com/vanilla/vanilla/pull/3068